db/schema.rb

```diff
@@ -526,7 +526,6 @@
   create_table "ucas_matches", force: :cascade do |t|
     t.bigint "candidate_id", null: false
     t.json "matching_data"
-    t.string "matching_state"
```

Comment:
Can we do this migration in a separate PR to be deployed later? We should change the code so it doesn't depend on the column in this PR, and remove the old column separately

Reply:
Seperate PR as suggested: https://github.com/DFE-Digital/apply-for-teacher-training/pull/3621